sdk/ml/azure-ai-ml/azure/ai/ml/operations/_schedule_operations.py

```diff
@@ -327,7 +327,7 @@ def _resolve_monitor_schedule_arm_id(  # pylint:disable=too-many-branches,too-ma
             error_messages = []
             if not signal.target_dataset or not signal.baseline_dataset:
                 # if there is no target dataset, we check the type of signal
-                if signal.type == MonitorSignalType.DATA_DRIFT or signal.type == MonitorSignalType.DATA_QUALITY:
+                if signal.type in {MonitorSignalType.DATA_DRIFT, MonitorSignalType.DATA_QUALITY}:
```

Comment:
nit: why curly braces instead of brackets for a list/array?

Reply:
The message that pylint emits suggests using a set if the type is hashable:

`Consider merging these comparisons with 'in' by using '%s %sin (%s)'. Use a set instead if elements are hashable.`

https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/consider-using-in.html